packages/shared/src/components/MainLayout.tsx

```diff
@@ -202,7 +201,6 @@ function MainLayout({
       {customBanner}
       {isBannerAvailable && <PromotionalBanner />}
       <InAppNotificationElement />
-      <LazyModalElement />
```

Comment:
Can you share what was the motivation for moving this element?

Reply:
When creating the separate modal component for the video we show on referral page and trying to use with the LazyModal setup, realised that the modal didn't work. This because join page is not wrapped by the MainLayout component. 
In order for every page to be able to render the modal using LazyModal setup this needed to be higher up in the hierarchy so got moved to _app.tsx where it will work for every page, no matter if they use main layout or not.

I'll add this in the PR description as well when I finish all the changes :D

Reply:
I see. Thank you for the clarification! That indeed would not work inside just the MainLayout. Sorry I reviewed way earlier, just a bit excited about this 😆